CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ cmake_minimum_required(VERSION 3.16)
 project(swift-atomics
   LANGUAGES C Swift)
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
```

Comment:
otherwise it tries to find `cmake/modules` in the root of the player directory

Reply:
proposed this change to the official repo already
https://github.com/apple/swift-atomics/pull/52

Reply:
LGTM